src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -23,11 +21,12 @@ using namespace nbl::ui;
 
 namespace nbl::ext::imgui
 {
-	smart_refctd_ptr<IGPUDescriptorSetLayout> UI::CreateDescriptorSetLayout()
+	smart_refctd_ptr<IGPUDescriptorSetLayout> UI::createDescriptorSetLayout()
 	{
-		nbl::video::IGPUDescriptorSetLayout::SBinding bindings[] = {
+		nbl::video::IGPUDescriptorSetLayout::SBinding bindings[] = 
+		{
 			{
-				.binding = 0,
+				.binding = 0u,
```

Comment:
as the user to provide a descriptor set layout, also as what binding Ix holds the textures we'll index

Reply:
resolved